Loop/Managers/DeviceDataManager.swift

```diff
@@ -474,9 +474,11 @@ private extension DeviceDataManager {
             alertManager?.addAlertSoundVendor(managerIdentifier: pumpManager.managerIdentifier,
                                                     soundVendor: pumpManager)
             
-            if pumpManager.status.deliveryIsUncertain && uncertainDeliveryAlert == nil {
+            deliveryUncertaintyAlertManager = DeliveryUncertaintyAlertManager(pumpManager: pumpManager, rootViewController: rootViewController)
```

Comment:
While this is a nice refactor, separating responsibilities, it doesn't avoid my concern that this class now has a `rootViewController`....

Reply:
Yes, I tried to explain my reasoning above. Talking tomorrow after the check-in sounds good.